e2e-tests/src/cardano_cli.py

```diff
@@ -111,16 +152,90 @@ def generate_cross_chain_keys(self):
         blake2b.update(vkey_bytes)
         vkey_hash = blake2b.digest().hex()
 
-        logger.info(f"Cross chain signing key: {pkey_hex}")
-        logger.info(f"Cross chain verification key: {vkey_hex}")
-        logger.info(f"Cross chain verification key hash: {vkey_hash}")
+        logger.debug(f"Cross chain signing key: {pkey_hex}")
+        logger.debug(f"Cross chain verification key: {vkey_hex}")
+        logger.debug(f"Cross chain verification key hash: {vkey_hash}")
 
         return pkey, vkey_hex, vkey_hash
 
     def get_stake_key_hash(self, stake_key):
-        logger.info("Getting stake key hash")
+        logger.debug("Getting stake key hash...")
         cmd = f"{self.cli} latest stake-address key-hash --stake-verification-key {stake_key}"
         result = self.run_command.run(cmd)
         if result.stderr:
             logger.error(result.stderr)
         return result.stdout.strip()
+
+    def get_address_key_hash(self, payment_vkey):
+        logger.debug("Getting address key hash...")
+        cmd = f"{self.cli} latest address key-hash --payment-verification-key {payment_vkey}"
+        result = self.run_command.run(cmd)
+        if result.stderr:
+            logger.error(result.stderr)
+        return result.stdout.strip()
+
+    def generate_policy_id(self, script_file):
+        logger.debug("Generating policy id...")
+        cmd = f"{self.cli} latest transaction policyid --script-file {script_file}"
+        result = self.run_command.run(cmd)
+        if result.stderr:
+            logger.error(result.stderr)
+        return result.stdout.strip()
+
+    def mint_token(self, tx_in, address, lovelace, amount, token, policy_script_filepath):
```

Comment:
Maybe `build_mint_tx` because it only builds the transaction, like the function below.
"token" is redundant I think.

Reply:
Yes, I see another `mint_token` below that has same name but does execute transaction.